lib/backend/report_test.go

```diff
@@ -84,3 +84,21 @@ func TestBuildKeyLabel(t *testing.T) {
 		require.Equal(t, tc.masked, buildKeyLabel(tc.input, sensitivePrefixes))
 	}
 }
+
+func TestBuildLabelKey_SensitiveBackendPrefixes(t *testing.T) {
+	testCases := []struct {
+		input  string
+		masked string
+	}{
+		{"/tokens/1234-5678", "/tokens/******678"},
+		{"/usertoken/1234-5678", "/usertoken/******678"},
+		{"/access_requests/1234-5678", "/access_requests/******678"},
+
+		{"/webauthn/sessionData/scope/1234-5678", "/webauthn/sessionData"},
```

Comment:
i built a key as was used in prod and the `buildKeyLabel` caps the key parts to two parts as stated here in [comments](https://github.com/gravitational/teleport/commit/00a4de5a52843020f63d8a55d82bd2cb09ea4c45#diff-a640ddd8ed0d579e4e5a19f8812542435b20e9024a52599da8b7aeb93fbf8dc1R288)